src/projects/apps/auth/authentication.md

```diff
@@ -0,0 +1,54 @@
+## Authentication
+
+### App Teams
+
+App users exist within a container called an "App Team", and each App can have one App Team assigned to it. 
+
+An App Team is automatically generated when you set up a new app with authentication but it also gives you the option to assign an existing App Team to your app. This allows you to share preexisting user accounts between Apps, foregoing the need to have users re-register when using the new App. You can change the App Teeam at any point after the Apps creation.
+This is particularly useful in scenarios where you are developing several different apps for the same audience, you can create a single App Team and then use it across every other App that has the same target audience.
+
+Imagine you're developing a suite of internal developer Apps and they all have the same target audience, you can create the first App and this will initialise the App team. Then, for any future Apps you create which will be accessed by the same users, you can simply assign the original App Team when setting up the authentication.
+
+### Username & Password
+
+Username & Password authentication is the most basic form of authentication offered in an App, it will allow users to log into an App by a unique username and password that is stored in the platform.
+The authentication flow is all handled within the platform as is the list of App users and their information.
+Once a user is created, they can be assigned to one or more Groups which will authorize them to access resources and pages within the App.
+
+### OAuth2/OpenID
+
+The OAuth2 and OpenID frameworks are very similar to each other and both allow you to utilise another authentication provider (e.g. Microsoft or Google) for your Apps authentication. If you have ever used an app or website that gives you the option to log in with Google, LinkedIn or Microsoft then this is OAuth2 and OpenID behind the scenes.
+In Apps, you can use any :docs-link[Identity Provider]{id="admin/identity_providers"} that you have set up in your platform as the authentication provider for your App.
+
+The main difference between OAuth2 and OpenID in Apps is that OpenID guarantees certain information can be retrieved for the user that has logged in such as username, name, date of birth, address, email and more.
```

Comment:
Probably worth saying that OpenId is a standard that implements OAuth2 which is why they are guaranteed to have those